cluster/openstack-heat/kubernetes-heat/fragments/configure-salt.yaml

```diff
@@ -32,6 +32,10 @@ write_files:
       password=$OS_PASSWORD
       region=$OS_REGION_NAME
       tenant-id=$OS_TENANT_ID
+      [LoadBalancer]
+      lb-version=$LBAAS_VERSION
+      subnet-id=$SUBNET_ID
```

Comment:
Ok, #29726 has merged.  I think we should make sure we default lb-version to `""` (empty string / unspecified) here, and let the autodetection code do its thing (still with the option for users to override if they need to).